src/BucketList.js

```diff
@@ -1,15 +1,15 @@
 import React from "react";
 import BucketListItem from "./BucketListItem";
 
-const BucketList = ({ bucketList, onRemoveBucket }) => {
+const BucketList = (props) => {
   return (
     <ul>
-      {bucketList.map((bucket) => (
+      {props.bucketList.map((item) => (
         <BucketListItem
-          key={bucket.id}
-          id={bucket.id}
-          title={bucket.title}
-          onRemoveBucket={onRemoveBucket}
+          key={item.id}
+          id={item.id}
+          title={item.fields.Title}
```

Comment:
this update should be implemented in src/TodoListItem.js. Please look at the instructions one more time (the last two lines).